build/media/libraries/redcore/media/redcore/js/jquery.searchtools.js

```diff
@@ -1,418 +0,0 @@
-
-			if (self.activeDirection.toUpperCase() == 'ASC')
-			{
-				newDirection = 'DESC';
-			}
-
-			self.activeDirection = newDirection;
-			self.activeOrder  = self.activeColumn + ' ' + newDirection;
-
-			self.updateFieldValue(self.orderField, self.activeOrder);
-		},
-		createOrderField: function () {
-
-			var self = this;
-
-			if (!this.orderField.length)
-			{
-				this.orderField = $('<input>').attr({
-					'type': 'hidden',
-					'id': 'js-stools-field-order',
-					'class': 'js-stools-field-order',
-					'name': self.options.orderFieldName,
-					'value': self.activeOrder + ' ' + this.activeDirection
-				});
-
-				this.orderField.appendTo(this.theForm);
-			}
-
-			// Add missing columns to the order select
-			if (this.orderField.is('select'))
-			{
-				this.orderCols.each(function(){
-					var value     = $(this).attr('data-order');
-					var name      = $(this).attr('data-name');
-					var direction = $(this).attr('data-direction');
-
-					if (value.length)
-					{
-						value = value + ' ' + direction;
-
-						var option = self.findOption(self.orderField, value);
-
-						if (!option.length)
-						{
-							var option = $('<option>');
-							option.text(name).val(value);
-
-							// If it is the active option select it
-							if ($(this).hasClass('active'))
-							{
-								option.attr('selected', 'selected');
-							}
-
-							// Append the option an repopulate the chosen field
-							self.orderField.append(option);
-						}
-					}
-
-				});
-
-				this.orderField.trigger('liszt:updated');
-			}
-
-			this.activeOrder  = this.orderField.val();
-		},
-		updateFieldValue: function (field, newValue) {
-			var self = this;
-			var type = field.attr('type');
-
-			if (type === 'hidden' || type === 'text')
-			{
-				field.attr('value', newValue);
-			}
-			else if (field.is('select'))
-			{
-				// Select the option result
-				var desiredOption = field.find('option').filter(function () { return $(this).val() == newValue; });
-
-				if (desiredOption.length)
-				{
-					desiredOption.attr('selected', 'selected');
-				}
-				// If the option does not exist create it on the fly
-				else
-				{
-					var option = $('<option>');
-					option.text(newValue).val(newValue);
-					option.attr('selected','selected');
-
-					// Append the option an repopulate the chosen field
-					field.append(option);
-				}
-
-				// Trigger the chosen update
-				if (self.chosenSupport) {
-					field.trigger('liszt:updated');
-				}
-			}
-		},
-		findOption: function(select, value) {
-			return select.find('option').filter(function () { return $(this).val() == value; });
-		}
-	};
-
-	// A really lightweight plugin wrapper around the constructor,
-	// preventing against multiple instantiations
-	$.fn[pluginName] = function (options) {
-		return this.each(function () {
-			if (!$.data(this, "plugin_" + pluginName)) {
-				$.data(this, "plugin_" + pluginName, new Plugin(this, options));
-			}
-		});
-	};
-
-})(jQuery, window, document);
```

Comment:
Even this file is not anymore in new installation, if you update redCORE this file remains:

<img width="845" alt="screen shot 2015-11-30 at 10 43 44" src="https://cloud.githubusercontent.com/assets/1375475/11468380/4dacce50-974f-11e5-858d-eefbcb92f4f3.png">

We would need a php script to be executed on update to remove it.